clients/src/main/java/org/apache/kafka/clients/consumer/internals/SubscriptionState.java

```diff
@@ -111,7 +108,7 @@ public SubscriptionState(LogContext logContext, OffsetResetStrategy defaultReset
      *
      * @return The current assignment Id
      */
-    public int assignmentId() {
```

Comment:
@hachikuji if you don't mind I adjusted the visibility where it could be reduced.